web/src/components/signup-form-google.js

```diff
@@ -60,59 +60,59 @@ class SignupFormGoogle extends Component {
     `;
 
     // Receive state values for mutation use.
-    const {id, email, firstName, lastName} = this.state
+    const {id, email, firstName, lastName} = this.state;
 
     return (
-      <div>
-        {/* A google log in button component. */}
-        <GoogleLogin
-            clientId={clientId}
-            buttonText="Google Log-in"
-            onSuccess={this.responseGoogle}
-            onFailure={this.responseFail}
-        >
-          <p style={{
-            width: '228px', height: '28px',
-            fontWeight: 'bold', fontStretch: 'normal',
-            lineHeight: '30px',
-            color: '#4a4a4a', fontFamily: 'Roboto', marginBottom: '0',
-          }}>
-            Sign up with Google
-          </p>
-        </GoogleLogin>
+        <div>
+          {/* Sends a mutation to the server. */}
+          <Mutation
+              mutation={SIGNUP_MUTATION}
+              variables={{
+                googleId: this.state.id,
+                email: this.state.email,
+                firstName: this.state.firstName,
+                lastName: this.state.lastName,
+              }}
+              onCompleted={
+                (data) => {
+                  const {success, message, token} = data.signUpWithGoogle;
+                  if (success) {
+                    document.cookie = 'token=' + token;
+                    this.props.history.push('/');
+                  } else {
+                    window.alert('Signup failed ... please contact admin');
```

Comment:
break line at 80 chars.

Reply:
I think it's under 80 chars.. What do you mean..?